### DISCUSSION TITLE
PowerOnState<x> for multi relay devices like the shelly 2.5PM

### DISCUSSION THREAD

**Discussion by brjhaverkamp** [arendst/Tasmota#22113] [Ideas]
### PROBLEM DESCRIPTION

 I use a Shelly 2.5PM. Relay one is used to measure the power usage towards my EV. There is no button attached.  The second relay is used  to switch the light in the carport. 
I noticed that the relay to the EV sometime switches. I assume ghost switches. So I want to fix that relay to always on. I don't want to run the risk that I get to my car in the morning and it hasn't charged because of a (ghost) switch switched the charger off. The carports lights need to stay switchable, of course.

In my previous setup I had a Shelly 1PM to measure the power of the EV and ensured it to be always on with PowerOnState 4

I would like to propose to extend PowerOnState to PowerOnState<x> and be able to set it for Relay1 and Relay2 independently.

### REQUESTED INFORMATION

- [ x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ x] Device used (e.g., Sonoff Basic):Shelly 2.5PM
- [ x] Tasmota binary firmware version number used: 14.2.0
  - [ x] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: FTDI
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:

**Comment by barbudor** [comment]
Having the ability to store PowerOnState (3-bits) for each possible relay (32) would be a huge consumption of Settings space

You just have to set the GPIO to "OUTPUT HIGH" and you're done (of course that will shift your relays indexes: 2 -> 1)
